content/applications/general/developer_mode.rst

```diff
@@ -4,76 +4,80 @@
 Developer Mode (debug mode)
 ===========================
 
-The developer mode (or debug mode) gives you access to extra and advanced tools.
+The developer mode (or debug mode) unlocks access to extra and advanced tools.
 
 Activate through the Settings
 =============================
 
-Go to :menuselection:`Settings --> Activate the developer mode`.
+Go to :menuselection:`Settings --> General Settings --> Developer Tools` and click on
```

Comment:
Add `The debug mode can be activated in the Odoo database settings.` before the sentence `` Go to...developer mode`. ``